ec/internal/planmodifiers/use_state_unless_template_changed.go

```diff
@@ -63,6 +64,15 @@ func (m useStateForUnknownUnlessMigrationIsRequired) PlanModifyInt64(ctx context
 func (m useStateForUnknownUnlessMigrationIsRequired) UseState(ctx context.Context, configValue attr.Value, plan tfsdk.Plan, state tfsdk.State, planValue attr.Value, stateValue attr.Value) (bool, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
+	var parentResState attr.Value
+	diags.Append(state.GetAttribute(ctx, path.Root(m.resourceKind), &parentResState)...)
+
+	resourceIsBeingCreated := parentResState.IsNull()
+
+	if resourceIsBeingCreated {
+		return false, nil
```

Comment:
```suggestion
		return false, diags
```
The same should be applied down the lines in all places where currently `nil` is returned.

Reply:
I mean we should return `diags` also in lines 77, 81 and 86.

Reply:
Sorry if I failed to make myself clear earlier.

Reply:
mm should I only add it here? why the others doesn't have it

Reply:
maybe is better to check and return, this looks a bit confusing